build/docker/intel-fpga-initcontainer.Dockerfile

```diff
@@ -75,7 +73,8 @@ RUN echo "{\n\
 ARG TOYBOX_VERSION="0.8.8"
 ARG TOYBOX_SHA256="2bed6bb9edd5a249023103cf0402a835b0e53d10304a263f6f1e77a8aa49a898"
 ARG ROOT=/install_root
-RUN apt update && apt -y install musl musl-tools musl-dev
+RUN apt-get update && apt-get --no-install-recommends -y install musl musl-tools musl-dev
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
```

Comment:
Just curious, what this is for?

Reply:
https://github.com/hadolint/hadolint/wiki/DL4006